docs/src/boilerplate.md.ejs

```diff
@@ -29,6 +29,8 @@ If you omit the "catch" section, errors may not be visible.
 
 ### Parameters
 
+The RippleAPI constructor takes an object with the following options:
```

Comment:
Maybe add "optionally" since it can be omitted. And we could point out that if omitted, only offline operations like "sign" and "computeLedgerHash" will be allowed.